src/main/java/com/launchcode/recipeproject/models/Ingredient.java

```diff
@@ -12,7 +12,7 @@
  */
 
 @Entity
-public class Ingredient extends AbstractEntity{
+public abstract class Ingredient extends AbstractEntity{
```

Comment:
Did you mean to make this an abstract class? abstract classes are classes which cannot have individual instances of objects created from them, which we need to be able to do.

Reply:
fixed